src/mon/Monitor.cc

```diff
@@ -3607,13 +3607,12 @@ void Monitor::handle_command(MonOpRequestRef op)
       }
       rdata.append(ds);
     } else if (prefix == "health") {
-      string plain;
-      healthmon()->get_health_status(detail == "detail", f.get(), f ? nullptr : &plain);
+      healthmon()->get_health_status(detail == "detail", f.get(), f ? nullptr : &ds);
```

Comment:
this does not compile. `ds` is a `stringstream`, while this parameter is supposed to be a `string*`.

Reply:
ops, I should compile first.